pkg/operator/csi/controller/credentials/controller.go

```diff
@@ -0,0 +1,91 @@
+package credentials
+
+import (
+	"context"
+	"time"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+
+	opv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/resource/resourcemerge"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+// Controller is a simple controller that maintains a CredentialsRequest static manifest.
+// It uses unstructured.Unstructured as currently there's no API type for this resource.
+type Controller struct {
```

Comment:
kube project standard realized that naming everything controller.go did bad things to glog output.  Please name your file something more specific even if you wish to keep this name (and I recommend making the struct name more specific, but that isn't externally visible)